StatBot/StatBot/Services/ConfigService.cs

```diff
@@ -6,11 +6,13 @@ namespace StatBot.Services
 {
     public class ConfigService
     {
-        private Config _config;
+        public static Config Config { get; set; }
```

Comment:
I think it's somewhat of a code smell to have a variable name identical to the type. If the type has any static properties, you will have a pain tryna reference them.